docs/tutorials/simple-date-clock-application.adoc

```diff
@@ -0,0 +1,435 @@
+= A simple date-clock application: tutorial cloud-harness
+
+In this small tutorial, we will see different aspects about the developement of applications with cloud harness through the developement from scratch of a small webapp that fetch information from a server on a regular basis.
+
+Cloud harness allows you to quickly setup app that needs to be ran in a kubernete cluster.
+It generates the initial files and folders for your project depending on some templates tacking different aspects of your app depending on your requirements, __e.g.__, for a webapp project, it generates the frontend initial files for ReactJS and the initial Flask files for the backend.
+For the API part, cloud harness relies on OpenAPI 3 to deal with the endpoints/model description.
+
+The different aspects that will be covered here are:
+
+* how to install cloud-harness;
+* how to quickly setup a first version of a local cluster using minikube;
+* how to build and locally deploy the projects from the repository;
+* how to bootstrap a new app, build it and run it;
+* how to modify/update the app, built it and run it again.
+
+== The tools you need
+
+Before installing everything, please be sure you have the following tools installed on your machine:
+
+* `python`
+* `minikube` -- the local mini cluster
+* `kubectl` -- to help controlling the cluster
+* `helm` -- to deal with Kubernete "packages" (helm is basically a package manager for Kubernete)
+* `skaffold` -- to build/deploy/run the apps
+
+
+== Install cloud-harness (if it is not yet done)
+
+The installation is pretty straight forward.
+Cloud-harness is coded in Python, consequently, it's always better to create a local virtualenv dedicated to the project to avoid messing with your system dependencies.
+As shown in the snippet above, we will clone the repository, change directory inside the freshly clone directory and create a virtualenv inside (for this tutorial, the Python's version used is CPython 3.10.6).
+
+.Cloning and installing cloud-harness
+[source,bash]
+----
+git clone git@github.com:MetaCell/cloud-harness.git
```

Comment:
all good here, just let's think on what happens in similar cases where the information can become outdated. Maybe using links can be appropriate in some contexts even if it breaks the flow?

Reply:
Perhaps then a link toward the git repo and inviting the readers to clone the repository themselves? Indeed, tt cut a little bit the flow, but at the end, it's only the beginning, it wouldn't be so hard to get back into it :)

Reply:
Ok, I tried something on this, but I'm unsure it's relevant... There is apparently no way or referencing the current repository as a auto link or through a special metavariable with github. I can create a macro otherwise in a common file that can be included, but it will only move the issue elsewhere. I added one sentence before, but I don't know exactly how to modify it after so it's relevant.

Reply:
Relative in-repo links work for markdowns, I expect the same to work here

Reply:
Indeed, relative URL works, but I wasn't sure I would reference directly the root of the repository. It turns out I can! Yesterday I failed trying it, but today is another day and it works :). Thanks @filippomc